go/libkb/log_send.go

```diff
@@ -108,34 +108,46 @@ func (l *LogSendContext) post(status, kbfsLog, svcLog, desktopLog, updaterLog, s
 }
 
 func tail(log logger.Logger, filename string, numLines int) string {
+	linesPrev := tailLines(log, filename+".1", numLines)
```

Comment:
@gabriel the previous log filenames do not end with ".1":

rw-------   1 patrick  staff   49871946 Jan 13 09:29 keybase.kbfs.log
-rw-------   1 patrick  staff  134217757 Jan 11 15:34 keybase.kbfs.log-20170111T144151-20170111T153435
-rw-------   1 patrick  staff  134217831 Jan 11 15:45 keybase.kbfs.log-20170111T153435-20170111T154554
-rw-------   1 patrick  staff  114687913 Jan 13 09:35 keybase.service.log
-rw-------   1 patrick  staff  134217780 Dec 22 16:04 keybase.service.log-20161222T095432-20161222T160420
-rw-------   1 patrick  staff  134228946 Jan  9 21:36 keybase.service.log-20170109T124051-20170109T213655
-rw-r--r--   1 patrick  staff     739485 Nov 23 16:10 keybase.start.log
-rw-r--r--   1 patrick  staff       1789 Jun 13  2016 keybase.testing.log
-rw-r--r--   1 patrick  staff    7740573 Jan 12 16:45 keybase.updater.log